src/javaharness/java/arcs/api/StorageProxy.java

```diff
@@ -1,5 +1,7 @@
 package arcs.api;
 
+import arcs.crdt.CRDTCollection;
```

Comment:
Nothing uses this yet?

Reply:
not yet, i added the import so that it builds with the hotreload script, and i'm adding it to the storage proxy in the upcoming pr.